[2025-01-26T22:18:07.739+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T22:18:07.794+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_gener_data.stg-load scheduled__2025-01-26T22:17:00+00:00 [queued]>
[2025-01-26T22:18:07.815+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_gener_data.stg-load scheduled__2025-01-26T22:17:00+00:00 [queued]>
[2025-01-26T22:18:07.815+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-26T22:18:07.862+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): stg-load> on 2025-01-26 22:17:00+00:00
[2025-01-26T22:18:07.880+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2021) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-26T22:18:07.885+0000] {standard_task_runner.py:72} INFO - Started process 2025 to run task
[2025-01-26T22:18:07.895+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_gener_data', 'stg-load', 'scheduled__2025-01-26T22:17:00+00:00', '--job-id', '475', '--raw', '--subdir', 'DAGS_FOLDER/DAG_test_data.py', '--cfg-path', '/tmp/tmpnrh5cv94']
[2025-01-26T22:18:07.907+0000] {standard_task_runner.py:105} INFO - Job 475: Subtask stg-load
[2025-01-26T22:18:08.056+0000] {task_command.py:467} INFO - Running <TaskInstance: load_gener_data.stg-load scheduled__2025-01-26T22:17:00+00:00 [running]> on host a664bcf281bf
[2025-01-26T22:18:08.703+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_gener_data' AIRFLOW_CTX_TASK_ID='stg-load' AIRFLOW_CTX_EXECUTION_DATE='2025-01-26T22:17:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-26T22:17:00+00:00'
[2025-01-26T22:18:08.705+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T22:18:08.806+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:08.880+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-26T22:18:08.914+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T22:18:08.915+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_gener_data, task_id=stg-load, run_id=scheduled__2025-01-26T22:17:00+00:00, execution_date=20250126T221700, start_date=20250126T221807, end_date=20250126T221808
[2025-01-26T22:18:08.999+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-26T22:18:09.080+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-26T22:18:09.085+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
